lib/galaxy/job_metrics/__init__.py

```diff
@@ -212,7 +212,7 @@ def collect_properties(self, job_id, job_directory):
                 if properties:
                     per_plugin_properties[plugin.plugin_type] = properties
             except Exception:
-                log.exception("Failed to collect job properties for plugin %s", plugin)
+                log.warning("Failed to collect job properties for plugin %s", plugin)
```

Comment:
```suggestion
                log.exception("Failed to collect job properties for plugin %s", plugin)
```
There could always be things happening we can actually fix.